media_cdn/snippets.py

```diff
@@ -149,3 +157,38 @@ def sign_cookie(url_prefix: str, key_name: str, base64_key: str, expiration_time
             policy=policy, signature=signature)
     return signed_policy
 # [END mediacdn_sign_cookie]
+
+
+# [START mediacdn_sign_token]
+def sign_token(url_prefix: str, base64_key: bytes, algo: str, expiration_time: datetime.datetime = None) -> bytes:
+    """Gets the Signed URL Suffix string for the Media CDN' Short token URL requests.
+
+    Args:
+        url_prefix: URL prefix to sign as a string.
+        base64_key: Secret key as a base64 encoded string.
+        algo: Algorithm can be either `SHA1` or `SHA256`.
+        expiration_time: Expiration time as a UTC datetime object.
+
+    Returns:
+        Returns the Signed URL appended with the query parameters based on the
+        specified URL prefix and configuration.
+    """
+    output = b"URLPrefix=" + base64.standard_b64encode(url_prefix.encode("utf-8"))
```

Comment:
This should be base64.urlsafe_b64decode like the other code samples.

Reply:
Understood! Will update it in the next commit.